package.json

```diff
@@ -33,5 +33,8 @@
     "npm": ">=7.0.0",
     "node": ">=14.0.0"
   },
-  "packageManager": "npm@8.10.0"
+  "packageManager": "npm@8.10.0",
+  "dependencies": {
+    "json2ts": "^0.0.7"
```

Comment:
Should this be hanging around here?

Reply:
It's mildly frustrating to me that the `npm add` command doesn't respect workspaces by default :shrug: